splash/browser_tab.py

```diff
@@ -870,6 +870,23 @@ def _post_mouse_event(self, type, button, x, y):
         event = QMouseEvent(type, point, q_button, buttons, modifiers)
         QApplication.postEvent(self.web_page, event)
 
+    def send_keys(self, text, key_type=None):
+        """
+        Send key events to webpage
+        :param text string to be sent as key strokes
+        :param key_type string representing the pressed key
+        :return: None
+        """
+        key_type = key_type or 'unknown'
+        # http://doc.qt.io/qt-5/qt.html#Key-enum
+        key_type = getattr(Qt, 'Key_%s' % key_type, Qt.Key_unknown)
+        self._send_keys(text, key_type)
+
+    def _send_keys(self, text, key_type):
+        modifiers = QApplication.keyboardModifiers()
+        event = QKeyEvent(QEvent.KeyPress, key_type, modifiers, text)
+        QApplication.postEvent(self.web_page, event)
```

Comment:
Yeah, it may go out of control pretty soon: if we want to closely emulate key presses we'd have to add keyboard layouts support to match keys pressed and unicode values produced. That'd be a nice feature, but that's too much for now.